doc/api/errors.md

```diff
@@ -800,6 +800,12 @@ An attempt was made to initiate operations on a watcher returned by
 
 An attempt was made to add more headers after the headers had already been sent.
 
+<a id="ERR_HTTP_INCOMING_SOCKET_ENCODING"></a>
+### ERR_HTTP_INCOMING_SOCKET_ENCODING
+
+An attempt was made to manipulate the encoding of an incoming request packet.
+This is not allowed (RFC2616).
```

Comment:
Suggestion: `This is forbidden by [RFC 2616](https://www.ietf.org/rfc/rfc2616.txt)`.